utilities/pulumi_state_splitter/tests/test_stored_state.py

```diff
@@ -0,0 +1,41 @@
+"""Testing `pulumi_state_splitter.stored_state`."""
+
+import unittest
+
+import typeguard
+
+with typeguard.install_import_hook("pulumi_state_splitter"):
+    import pulumi_state_splitter.stored_state
+
+
+class TestStackName(unittest.TestCase):
```

Comment:
You might add a test case with invalid format that doesn't specify any `/` for completeness.

Reply:
Done in e84e7072745323c1878141f78ded91a3376c55ec .